tools/run_tests/run_build_statistics.py

```diff
@@ -243,6 +243,6 @@ def _process_build(json_url, console_url):
     rows = [big_query_utils.make_row(build_number, build_result)]
     if not big_query_utils.insert_rows(bq, _PROJECT_ID, _DATASET_ID, build_name, 
                                        rows):
-      print '====> Error uploading result to bigquery.'
+      print('====> Error uploading result to bigquery.')
```

Comment:
Missing `from __future__ import print_function` in this file (should be after module doc string and before "ordinary" imports; see elsewhere around the codebase for example to follow).